components/affix/index.en-US.md

```diff
@@ -0,0 +1,21 @@
+---
+category: Components
+type: Other
+english: Affix
+---
+
+Make element sticky in view.
+
+## When to use
+
+When you scroll in a long page, some content need to presist in view. It is common for menus and actions.
+
+Affix should be used with caution when view window is limited, avoiding cover other content in page.
```

Comment:
Please note that Affix should not cover other content in page, especially when the viewport is small.